frontend/src/components/ui/file-list/file-list.ts

```diff
@@ -0,0 +1,43 @@
+import { css, html } from "lit";
+import { customElement, queryAssignedElements } from "lit/decorators.js";
+
+import { TailwindElement } from "@/classes/TailwindElement";
+
+@customElement("btrix-file-list")
+export class FileList extends TailwindElement {
```

Comment:
This file is split from `frontend/src/components/ui/file-list.ts` without changes to the component.